### DISCUSSION TITLE
sonoff t1 2 gang one button always on and one button normal switch

### DISCUSSION THREAD

**Discussion by mpuff** [arendst/Tasmota#19343] [Q&A]
hello,

i have a sonoff t1 2 gang, on toggle 1 i want normal use of the switch, with on off at pressing and mqtt, the second button should
be always on and only send mqtt message if it is on or of, also the light should normal goes on at on and off at off, i want to control the light via smartphone rule, because the light is a zigbee device and should be always on.
what command do I need to set in tasmota to do that?

i am using the latest v13.1 tasmota version.
kind regards,
mpuff

**Comment by barbudor** [comment]
I understand that by "the second button should be always on" you mean the relay.

You have different solutions to detach the button from the relay
Have you read https://tasmota.github.io/docs/Buttons-and-Switches/#button ?

The simplest way would be to use rules to send the Zigbee on/off command:
```
rule1 on Button1#State do publish topic/of/your/zigbeegw xxxx %value% endon
```
For a button, single press will always send a TOGGLE (2) value

You then need to insure that the Relay2 is always on:
Replace the GPIO for relay by "OUTPUT_HI" and your relay will always be ON
